tests/acceptance/features/apiSpacesShares/shareOperations.feature

```diff
@@ -365,17 +366,20 @@ Feature: sharing
       | path        | /FOLDER       |
       | permissions | <permissions> |
       | password    |               |
-    Then the HTTP status code should be "200"
+    Then the HTTP status code should be "<http_status_code>"
     And the OCS status code should be "400"
     And the OCS status message should be "missing required password"
     Examples:
-      | permissions |
-      | change      |
-      | create      |
+      | ocs_api_version | permissions | http_status_code |
+      | 1               | change      | 200              |
+      | 2               | change      | 400              |
+      | 1               | create      | 200              |
+      | 2               | create      | 400              |
```

Comment:
Http status code are a little bit different depending on the ocs version idk if this is expected or not

Reply:
AFAIK, we have these differences between v1 and v2 in oc10 as well